Cppcheck issue un Windows Launcher

**Describe the Bug**
The changes introduced in #1585 did raise new cppcheck failure:
```
src/webots/launcher/launcher.c:40:90: error: Null pointer dereference [nullPointer]
290    snprintf(message, sizeof(message), "%s failed with error %ld.\n%s%s%s", function, e, m, i, lf);
```

--- a/src/webots/launcher/launcher.c
+++ b/src/webots/launcher/launcher.c
@@ -37,7 +37,7 @@ static int fail(const char *function, const char *info) {
     char message[1024];
     const char *lf = info ? "\n" : "";
     const char *i = info ? info : "";
-    snprintf(message, sizeof(message), "%s failed with error %ld.\n%s%s%s", function, e, m, i, lf);
+    snprintf(message, sizeof(message), "%s failed with error %lu.\n%s%s%s", function, e, m, i, lf);
     LocalFree(m);
 #ifdef WEBOTSW
     MessageBox(NULL, message, "Webots launcher error", MB_ICONERROR | MB_OK);

--- a/src/webots/launcher/launcher.c
+++ b/src/webots/launcher/launcher.c
@@ -37,8 +37,8 @@ static int fail(const char *function, const char *info) {
     char message[1024];
     const char *lf = info ? "\n" : "";
     const char *i = info ? info : "";
-    // cppcheck-suppress-nullPointer
-    snprintf(message, sizeof(message), "%s failed with error %lu.\n%s%s%s", function, e, m, i, lf);
+    // cppcheck-suppress nullPointer
+    snprintf(message, sizeof(message), "%s failed with error %ld.\n%s%s%s", function, e, m, i, lf);
     LocalFree(m);
 #ifdef WEBOTSW
     MessageBox(NULL, message, "Webots launcher error", MB_ICONERROR | MB_OK);

--- a/src/webots/launcher/launcher.c
+++ b/src/webots/launcher/launcher.c
@@ -38,7 +38,7 @@ static int fail(const char *function, const char *info) {
     const char *lf = info ? "\n" : "";
     const char *i = info ? info : "";
     // cppcheck-suppress nullPointer
-    snprintf(message, sizeof(message), "%s failed with error %ld.\n%s%s%s", function, e, m, i, lf);
+    snprintf(message, sizeof(message), "%s failed with error %lu.\n%s%s%s", function, e, m, i, lf);
     LocalFree(m);
 #ifdef WEBOTSW
     MessageBox(NULL, message, "Webots launcher error", MB_ICONERROR | MB_OK);

--- a/src/webots/launcher/launcher.c
+++ b/src/webots/launcher/launcher.c
@@ -37,6 +37,7 @@ static int fail(const char *function, const char *info) {
     char message[1024];
     const char *lf = info ? "\n" : "";
     const char *i = info ? info : "";
+    // cppcheck-suppress-nullPointer
     snprintf(message, sizeof(message), "%s failed with error %lu.\n%s%s%s", function, e, m, i, lf);
     LocalFree(m);
 #ifdef WEBOTSW

